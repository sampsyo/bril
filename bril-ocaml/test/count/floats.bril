@main {
  a: float = const 2;
  b: float = const 3;
  x: int   = const 4;
  y: bool  = eq x x;
  print a;
}
