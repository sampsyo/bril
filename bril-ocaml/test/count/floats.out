Ints: 1 Bools: 1 Floats: 2 Pointers: 0 
